.circleci/config.yml

```diff
@@ -42,7 +46,11 @@ jobs:
       - image: cimg/ruby:2.7.7
     steps:
       - checkout
-      - ruby/install-deps
+      - run:
+          name: Install dependencies
+          command: |
```

Comment:
can we create a generic component for bundle install and re-use it? Please check if this is possible with circle ci

Reply:
resolved